test/test.dart

```diff
@@ -399,9 +399,18 @@ String generateRandomRealmPath() {
 }
 
 final random = Random();
+String _getRandom(int len, String chars) => List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
+
 String generateRandomString(int len) {
-  const chars = 'abcdefghjklmnopqrstuvwxuz';
-  return List.generate(len, (index) => chars[random.nextInt(chars.length)]).join();
+  return _getRandom(len, 'abcdefghjklmnopqrstuvwxuz');
+}
+
+String generateRandomUnicodeString(int len) {
+  return _getRandom(len, 'uvwxuzфоо-барΛορεμლორემ植物החללجمعتsøren');
+}
+
+String generateRandomEmail(int len) {
+  return _getRandom(len, r"abcdefghjklmnopqrstuvwxuz!#$%&*+-'/=?^_`{|}~0123456789");
```

Comment:
This does not seem to generate an random email. It just generates a random String from what it seems allowed characters presumably for valid email address. 
I suggest we refactor this code to really return a random email address in full including the domain name and make this one invoke the generateRandomString

Reply:
also I don't think we need all the allowed symbols in an email. We just need to make sure we are using a different email in our tests.

Reply:
I refactored the methods as you suggested. I also added an options for including the allowed characters for email, because I think that it is good to have them in case some email validation issue appear in future.